### ISSUE TITLE
Selectable & Copy-able text in every cell

### ISSUE THREAD

**Issue by yotamgever** [deanshub/pivot2#10]
As a **viewer**, I would like to have the ability to select the cell text (or several cells), so I will be able to copy it outside the Sisense environment.
![image](https://cloud.githubusercontent.com/assets/4380713/22512973/663052de-e8a3-11e6-805a-ae8f64251232.png)

**Comment by yotamgever** [comment]
Data bars should be copied as well?

**Comment by deanshub** [comment]
referencing #13

**Comment by deanshub** [comment]
no, data bars and other visualizations should not be copied (they should maybe be supported with export to excel)

**Comment by orelse94** [comment]
time estimation 1 day
